src/cpu/x64/jit_avx512_core_amx_convolution.cpp

```diff
@@ -387,7 +387,7 @@ jit_avx512_core_amx_convolution_fwd_t::execute_forward_reduced_lowering(
                     limit_idx++;
                 p.ohb = limit_idx;
                 p.last_h = (oh + oh_step <= oh_e);
-                const int zp_owb = nstl::min(jcp.l_pad_blk, owb)
+                const size_t zp_owb = nstl::min(jcp.l_pad_blk, owb)
```

Comment:
If `nstl::min(jcp.l_pad_blk, owb)` will produce `int`, this change won't resolve an overflow.